src/constants/HIDmap.ts

```diff
@@ -5,6 +5,8 @@ export interface HidInfo {
   category: HIDCategory
   displayString: string
   webKeyId: string
+  whichID: number
+  locationID: number
```

Comment:
Added more data to the struct to fix other layouts while recording triggers

Reply:
Tweaked later in [643c014](https://github.com/WootingKb/wooting-macros/pull/194/commits/643c014ba8fdf17075af61d885d101f68b79c2c8)